app/views/static_pages/contributing/_hall_of_fame.erb

```diff
@@ -108,32 +108,25 @@
   ]
 %>
 
-<div class="hall-of-fame gradient">
-  <h2 class="hall-of-fame__title text-2xl">Hall of Fame</h3>
-  <p class="hall-of-fame__text">
+<div class="pt-16 gradient">
+  <h2 class="text-gold text-center font-bold pb-7 text-2xl">Hall of Fame</h3>
+  <p class="text-center w-4/5 sm:w-3/5 m-auto mb-5">
     The Hall of Fame members are the hardworking devs who have built The Odin Project into the site that you see before you now.
   </p>
 
-
-  <div class="container hall-of-fame__members">
-    <div class="row">
+  <div class="container py-14 mb-6">
+    <div class="grid grid-cols-12 gap-6 mb-12">
       <% members.each do |member| %>
-        <div class="col-lg-2 col-md-3 col-sm-4 col-xs-6 hall-of-fame__member">
+        <div class="col-span-4 sm:col-span-3 md:col-span-2">
           <%= link_to(member[:github_url], target: "_blank", rel: "noreferrer") do %>
-            <%= image_tag(member[:image], alt: member[:name], class: 'img-fluid') %>
+            <%= image_tag(member[:image], alt: member[:name]) %>
           <% end %>
         </div>
       <% end %>
     </div>
-  </div>
 
-  <div class="hall-of-fame__view-all">
-    <%= link_to 'View All Contributors', github_link('theodinproject/graphs/contributors'), class: 'camel-link mb-5', target: '_blank', rel: 'noreferrer' %>
+    <div class="text-center underline">
+       <%= link_to 'View All Contributors', github_link('theodinproject/graphs/contributors'), class: 'text-gold mb-6', target: '_blank', rel: 'noreferrer' %>
+    </div>
   </div>
-
-  <%= render 'shared/bottom_cta',
-    button: chat_button,
-    heading: 'Have a question?',
-    sub_heading: 'Chat with our friendly Odin community in our Discord chatrooms!'
-  %>
```

Comment:
Deleted this because it overlaps with the updated footer:

![image](https://user-images.githubusercontent.com/58506115/193447363-11a9a0ec-4484-4301-afad-64096d0de7ab.png)

This example is from the profile page, but same idea. Chat link only shows when logged in, though

Reply:
Makes complete sense to me, nice work!